templates/concert-ticket/templates/main.tpl

```diff
@@ -0,0 +1,173 @@
+        </paragraph>
+        <paragraph text-align="left" margin="5 5 0 0">
+            <text-chunk font="helvetica" font-size="{{$subFont}}">25.05.2021  7:30PM</text-chunk> 
+        </paragraph>
+        <line fit-mode="fill-width" position="relative" thickness= "2.0" margin="5 0 0 0"></line>
+    </division>
+{{end}}
+{{define "ticket-detail"}}
+{{$fieled := .Name}}
+{{$fontSize := .FontSize}}
+{{$fontName := .FontName}}
+    <table-cell indent="0">
+        <division margin="10 0 0 0">
+            <paragraph text-align="left" margin="0 0 0 0">
+                <text-chunk font="{{$fontName}}" font-size="{{$fontSize}}">{{$fieled}}</text-chunk>
+            </paragraph>
+        </division>
+    </table-cell>
+{{end}}
+<table columns="2" margin = "0 0 10 10" padding="0, 0, 30, 30" column-widths="0.20 0.80">
+    <table-cell vertical-align="top">
+        <image src="path('./templates/res/red-guitar.png')" width="100" height="160" margin="5 0 0 0"></image>
+    </table-cell>
+    <table-cell vertical-align="top">
+        <division>
+            <division margin="0 0 0 15">
+            {{template "header" dict "FontSize" 19 "SubFont" 11}}
+            </division>
+                <table columns="2" padding="0, 0, 0, 0" column-widths="0.3 0.7">
+                <table-cell>
+                    <table columns="1" margin="10 0 0 0">
+                        <table-cell>
+                            <division margin="0 0 0 0" padding="0, 0, 0, 0">
+                                <paragraph text-align="left" margin="0 0 0 10" line-height="1.1">
+                                    <text-chunk font="helvetica" font-size="10">E - ticket</text-chunk>
+                                </paragraph>
+                                <paragraph text-align="left" margin="2 0 0 10">
+                                    <text-chunk font="helvetica" font-size="10">{{.TicketNumber}}</text-chunk>
+                                </paragraph>
+                                <image src="qr-code" height="70" width="70" margin="5 0 0 10"></image>
+                            </division>
+                        </table-cell>
+                    </table>
+                </table-cell>
+                <table-cell>
+                    <table columns="2">
+                        {{range .Detail}}
+                        {{template "ticket-detail" dict "FontName" "helvetica" "FontSize" 11 "Name" .FieldName }}
+                        {{template "ticket-detail" dict "FontName" "helvetica" "FontSize" 11 "Name" .FieldValue }}
+                        {{end}}
+                    </table>
+                </table-cell>
+            </table>
+        </division>
+    </table-cell>
+</table>
+<table columns="2" margin="0 0 0 15">
+ <table-cell border-width-top="0.5" border-width-bottom="0.5" border-width-left="0.5" vertical-align="top" indent="0.5" border-style="single">
+    <division margin="0 0 15 15" padding="0, 0, 0, 0">
```

Comment:
No need to specify the padding. `0` is the default value. Also, the syntax of the attribute is not correct. The values should be separated by spaces.